api/resources/views/vendor/mail/html/message.blade.php

```diff
@@ -0,0 +1,27 @@
+@component('mail::layout')
```

Comment:
Do we need to export all those blade files? If there is an update, we'll need to update all manually.
Nothing is changed I guess to this file